# This message is for storing a single object detection
#  Raymond Kirk (Tunstill) Copyright (c) 2020
#  Email: ray.tunstill@gmail.com

rasberry_perception/RegionOfInterest roi # represents bounding box detections
rasberry_perception/SegmentOfInterest seg_roi # represents pixel segmentations (masks)
rasberry_perception/DetectionInfo info # information about the detection (score, class, label etc.)